Doc/Tutorial/chapter_uniprot.tex

```diff
@@ -57,16 +57,20 @@ \subsection{Parsing Swiss-Prot records}
 %TODO - Check the single quotes when printing record.description here:
 \begin{verbatim}
 >>> print(record.description)
-'RecName: Full=Chalcone synthase 3; EC=2.3.1.74; AltName: Full=Naringenin-chalcone synthase 3;'
```

Comment:
What was the ``myaccessionnumber`` value used earlier for this:

```
 >>> from Bio import ExPASy
 >>> handle = ExPASy.get_sprot_raw(myaccessionnumber)
```

```
 >>> from Bio import SwissProt
 >>> record = SwissProt.read(handle)
```

```
>>> print(record.description)
```

Can we make this into an online Tutorial doctest using the special comment line ``%doctest . internet``?